sphinx/source/docs/user_guide/jupyter.rst

```diff
@@ -48,21 +48,44 @@ multiple times in the input cell. The plots will display in order.
 JupyterLab
 ++++++++++
 
-To embed Bokeh plots in JupyterLab, you'll need the following two JupyterLab extensions:
+To use JupyterLab with Bokeh, you should at least use version 3.0 of JupyterLab.
+Enabling Bokeh visualizations in JupyterLab also requires the
+`jupyter_bokeh <https://github.com/bokeh/jupyter_bokeh>`_ extension to be
+installed.
 
-1. First install *jupyterlab-manager* with this command:
+After installing JupyterLab, you can use either ``pip`` or ``conda`` to install
+jupyter_bokeh:
 
-   .. code::
+.. panels::
 
-    jupyter labextension install @jupyter-widgets/jupyterlab-manager
+    Installing with ``conda``
+    ^^^^^^^^^^^^^^^^^^^^^^^^^
 
-2. Then install the *jupyter_bokeh* extension as follows:
+    Make sure you have either `Anaconda`_ or `Miniconda`_ installed. Use
+    this command to install jupyter_bokeh:
 
-   .. code::
+    .. code-block:: sh
 
-    jupyter labextension install @bokeh/jupyter_bokeh
+        conda install jupyter_bokeh
 
-The rest is the same as with classic notebooks above.
+    ---
+
+    Installing with ``pip``
+    ^^^^^^^^^^^^^^^^^^^^^^^
+
+    Use this command to install jupyter_bokeh:
+
+    .. code-block:: sh
+
+        pip install jupyter_bokeh
+
+For instructions on installing jupyter_bokeh with versions of JupyterLab
+older than 3.0, see the
+`README <https://github.com/bokeh/jupyter_bokeh/blob/main/README.md>`_ in the
+`GitHub repository of jupyter_bokeh <https://github.com/bokeh/jupyter_bokeh>`_.
```

Comment:
Another details: At least up until the present, the docs have always avoided inline URLs and used `.. _foo: <url>` links instead. If we are going to change it would be good to decide what the criteria is for using one or the other